c10/util/TypeTraits.h

```diff
@@ -130,6 +130,25 @@ struct is_type_condition<C, std::enable_if_t<std::is_same<bool, std::remove_cv_t
 template <class T>
 struct is_fundamental : std::is_fundamental<T> {};
 
+/// Strip the class from a method type
```

Comment:
We already have this functionality, see infer_function_traits in Metaprogramming.h